packages/@cdktf/provider-generator/lib/get/generator/models/resource-model.ts

```diff
@@ -103,10 +99,6 @@ export class ResourceModel {
     return this.terraformSchemaType === "provider";
   }
 
-  public get namespace(): ResourceNamespace | undefined {
-    return getResourceNamespace(this.provider, this.baseName);
-  }
-
```

Comment:
Wouldn't it be valuable to keep this field on a resource model? Where is the namespace living now?
I'd refrain from having it implicit in multiple places instead of explicitly queried from here 🤔

Reply:
Yeah we could put it there as well, but it works as is so I'd safe this refactoring for after 0.13 since it should not change any user-facing code.